client.go

```diff
@@ -25,3 +25,40 @@ func (c *client) metrics() (metrics, error) {
 	var res metrics
 	return res, c.rpcClient.Call(&res, "debug_metrics", true)
 }
+
+func (c *client) currentBlock() (string, error) {
+	var res string
+	return res, c.rpcClient.Call(&res, "eth_blockNumber")
+}
+
+func (c *client) syncingFake() (metrics, error) {
```

Comment:
I would call it `currentBlockMetrics` or something like that.

Reply:
k